### PR TITLE
Create the 'Search' App

### LINKED ISSUE #107: Create 'Search' App
Create the `Search' app. 

This app needs to provide the ability to search apartments by preferences.

### PR DESCRIPTION
**Pull request by AmitAharoni**
In this PR:
- Created the 'Search' App.
- Registered it as an app on our website.
- Created a very simple `search.html` file.

fix: #107 

NOTE:
On the next PRs we will provide the ability to search apartments by preferences.

### CODE REVIEW

File: search/templates/search/search.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (Micha546) [inline search/templates/search/search.html]: Maybe extend the base template @nadavsu made in main? we will have to do it anyways

File: search/urls.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
```
Comment (Micha546) [inline search/urls.py]: Can you please import it as `search_views`? it's how I called the views in seekers and apartments and I think its more clear if we will call all of them the same way.
Other than that looking great!

File: search/templates/search/search.html (hunk 1 above)
Comment (AmitAharoni) [reply search/templates/search/search.html]: In this PR I just created the `search` App. 
I will upgrade the app (including the html file) in the next PRs, so in order to keep the PR  simple, I chose to create it without the base template.

Anyway, I just added that as you asked. 
Done :)

File: search/urls.py (hunk 2 above)
Comment (AmitAharoni) [reply search/urls.py]: Done.

**Review by DanielMy777 — COMMENTED** [review]
Seems great 👍

**Review by simzacks — APPROVED** [review]

**Review by OrAsaf — APPROVED** [review]

**Pull request merged**